src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -84,5 +84,25 @@ public async Task<ActionResult> GetByIdAsync(Guid id)
                 }
             });
         }
+
+        [HttpPut]
+        [Route("{id}")]
+        public async Task<ActionResult> UpdateOrganisationByIdAsync(Guid id, UpdateOrganisationViewModel viewModel)
```

Comment:
`viewModel` isn't actually a view model here but there are currently inconsistencies and differences of opinion on this one so this is something we're planning to discuss tomorrow.

Reply:
I see. For me, in this instance a VM is a DTO representing the json we'd be getting / returning.
I don't mind renaming this, as long as its solution-wide, agreed upon and understood by the whole team. But more on this tomorrow.

I'll be leaving it as is for now, and based on the outcomes of tomorrow's meeting either change it or keep it.